test/SourceKit/CompileNotifications/diagnostics.swift

```diff
@@ -87,3 +87,42 @@
 // RUN: %sourcekitd-test -req=track-compiles == -req=syntax-map %s | %FileCheck %s -check-prefix=SYNTACTIC
 // SYNTACTIC: key.syntaxmap:
 // SYNTACTIC_NOT: key.notification: source.notification.compile-did-finish
+
+// RUN: %sourcekitd-test -req=track-compiles == -req=sema %S/Inputs/descriptive-error.swift -- -Xfrontend -enable-descriptive-diagnostics -Xfrontend -diagnostic-documentation-path -Xfrontend /educational/notes/path/prefix %S/Inputs/descriptive-error.swift | %FileCheck %s -check-prefix=DESCRIPTIVE
```

Comment:
> -req=track-compiles

This is an informational tool designed for e.g. logging, and is not used by the editor for producing live diagnostics.  The output from `-req=sema` itself ought to be what is tested here I think.  I suggest moving this test to the "Sema" directory and not using `track-compiles`.

Reply:
Thanks for explaining this, I misunderstood what "Compile Notifications" were in SourceKit